src/xmipp/libraries/reconstruction/tomo_filter_coordinates.h

```diff
@@ -0,0 +1,102 @@
+/***************************************************************************
+ *
+ * Authors:    Federico P. de Isidro-Gómez    fp.deisidro@cnb.csic.es (2021)
+ *
+ * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+ * 02111-1307  USA
+ *
+ *  All comments concerning this program package may be sent to the
+ *  e-mail address 'xmipp@cnb.csic.es'
+ ***************************************************************************/
+
+#ifndef __TOMO_FILTER_COORDINATES
+#define __TOMO_FILTER_COORDINATES
+
+
+#include <iostream>
+#include <core/xmipp_program.h>
+#include <core/xmipp_filename.h>
+#include <core/xmipp_image.h>
+#include <data/point3D.h>
+#include <core/metadata_vec.h>
+
+#define VERBOSE_OUTPUT
+//#define DEBUG_DIM
+
+
+class ProgTomoFilterCoordinates : public XmippProgram
+{
+
+public:
```

Comment:
keep all private, except what really needs to be public. Class attributes are typically never public

Reply:
Done